esphome/components/font/__init__.py

```diff
@@ -313,6 +336,7 @@ def _file_schema(value):
         cv.Required(CONF_ID): cv.declare_id(Font),
         cv.Required(CONF_FILE): FILE_SCHEMA,
         cv.Optional(CONF_GLYPHS, default=DEFAULT_GLYPHS): validate_glyphs,
+        cv.Optional(CONF_GLYPHSETS): validate_glyphsets,
```

Comment:
```suggestion
        cv.Optional(CONF_GLYPHSETS): cv.ensure_list(cv.one_of(*glyphsets.defined_glyphsets())),
```
In addition, if `glyphsets` is provided, the default for `glyphs` should be empty, I would think.

Reply:
Fully agreed on the former, that's awesome :)

On the latter point w.r.t. emptying `glyphs`... it's a good point, and I think I agree with you but maybe worth thinking it through. I think in at least latin-adjacent countries (like Greek) it would uncommon to not have Latin glyphs at all in the set. (There are no fonts that support Greek but not Latin, for example). One could obviously add `GF_Latin_Kernel` or `GF_Latin_Core` to their list, but maybe that's superfluous?